packages/deployments/contracts/contracts/core/connext/facets/BridgeFacet.sol

```diff
@@ -640,16 +647,10 @@ contract BridgeFacet is BaseConnextFacet {
     } else {
       // If there are no routers for this transfer, this `execute` must be a slow liquidity route; in which
       // case, we must make sure the transfer's been reconciled.
-      if (!reconciled) revert BridgeFacet__execute_notReconciled();
-    }
-
-    // Require that this transfer has not already been executed. If it were executed, the `transferRelayer`
-    // would have been set in the previous call (to enable the caller to claim relayer fees).
-    if (s.transferRelayer[transferId] != address(0)) {
-      revert BridgeFacet__execute_alreadyExecuted();
+      if (status != DestinationTransferStatus.Reconciled) revert BridgeFacet__execute_notReconciled();
```

Comment:
might as well make this whole block an `else if`

Reply:
ehh would require me to handle the else

Reply:
`else` would just `pass` - no decision there

but it's fine, nbd

Reply:
no you need it to throw if the status fails, not just do nothing (indicating pass of sanity checks)

Reply:
lol hold up. I think this
```solidity
if (pathLength != 0) {
  if (status != DestinationTransferStatus.None) revert BridgeFacet__execute_badFastLiquidityStatus();
} else if (status != DestinationTransferStatus.Reconciled) {
  revert BridgeFacet__execute_notReconciled();
}
```
is the equivalent of this
```solidity
if (pathLength != 0) {
  if (status != DestinationTransferStatus.None) revert BridgeFacet__execute_badFastLiquidityStatus();
} else {
  if (status != DestinationTransferStatus.Reconciled) revert BridgeFacet__execute_notReconciled();
}
```

In the first case, there is no `else` case needed. If the `else if` conditional is not met, then we just proceed.